<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatwin</class>
 <widget class="QWidget" name="chatwin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>741</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>741</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px;
    border-color: Silver;
    border-style: solid;
border-radius: 10px; </string>
  </property>
  <widget class="QWidget" name="meue_wid" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(80, 80, 80);
border-radius: 10px; 
border:none</string>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/res/icons/icon1.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>32</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="chat_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="man_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="date_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="game_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mus_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="home_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>559</y>
      <width>32</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="info_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pho_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="function_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="user_wid" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>331</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(222, 222, 222);
border:none;
border-radius: 0px; </string>
   </property>
   <widget class="QWidget" name="select_wid" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border: 1px;
border-right: 1px;
    border-color: Silver;
    border-style: solid;
border-radius: 10px;
</string>
    </property>
    <widget class="QLineEdit" name="select_line">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px;
border-radius:10px;
    border-color: Silver;
    border-style: solid;
background-color:rgb(222, 222, 222);
font-size:18px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

QPushButton{
background-color:rgb(222, 222, 222);
font-size:20px;
border-radius:8px;
}
QPushButton:hover{
	background-color: Silver;
}
QPushButton:pressed{
    background-color: rgb(175, 175, 175);
}
</string>
     </property>
     <property name="text">
      <string>＋</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="chat_list">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>81</y>
      <width>291</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <item>
     <property name="text">
      <string>聊天大厅</string>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英雄联盟</string>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>永劫无间</string>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>我的世界</string>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>原神</string>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="enter_btn">
   <property name="geometry">
    <rect>
     <x>992</x>
     <y>700</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
    background-color: rgb(227, 227, 227);
	color:green; 
	font-size: 18px;
}
QPushButton:hover{
	background-color: Silver;
    text-decoration: underline;
}
QPushButton:pressed{
    background-color: rgb(175, 175, 175);
}
</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="tool_wid" native="true">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>519</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom:0px;</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>459</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="emoji_btn">
       <property name="styleSheet">
        <string notr="true">font-size:22px;
border:none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cut_btn">
       <property name="styleSheet">
        <string notr="true">font-size:22px;
border:none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pic_btn">
       <property name="styleSheet">
        <string notr="true">font-size:22px;
border:none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="place_btn">
       <property name="styleSheet">
        <string notr="true">font-size:22px;
border:none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="file_btn">
       <property name="styleSheet">
        <string notr="true">font-size:22px;
border:none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zd_btn">
       <property name="styleSheet">
        <string notr="true">font-size:22px;
border:none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="voice_btn">
       <property name="styleSheet">
        <string notr="true">font-size:22px;
border:none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>1084</x>
     <y>10</y>
     <width>26</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
}
QPushButton:hover{
    background-color: rgb(219, 75, 75);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="min_btn">
   <property name="geometry">
    <rect>
     <x>1051</x>
     <y>10</y>
     <width>27</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border:none;
}
QPushButton:hover{
    background-color: rgb(219, 75, 75);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>751</width>
     <height>521</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>751</width>
     <height>521</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0 0 10px 0;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="name_wid" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>751</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);
border: 1px;
border-right: 1px;
    border-color: Silver;
    border-style: solid;
border-radius: 10px;</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>32</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
font-size:21px;</string>
      </property>
      <property name="text">
       <string>聊天大厅</string>
      </property>
     </widget>
     <widget class="QLabel" name="online">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>40</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image:url(:/res/icons/online.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>691</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>691</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>tool_wid</zorder>
  <zorder>close_btn</zorder>
  <zorder>min_btn</zorder>
  <zorder>user_wid</zorder>
  <zorder>meue_wid</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>enter_btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
